### ISSUE TITLE
[Mod Compat] "Chipped" Recipes Rendering Error (JEI Compat)

### ISSUE THREAD

**Issue by BgueC6595** [emilyploszaj/emi#763]
Please let me know if this should be reported to Chipped instead. Since it works fine with JEI, I thought I should report it here as a JEI compat issue.

Chipped recipes will say "Error Rendering"
![image](https://github.com/user-attachments/assets/5956070a-c7be-400e-b582-f41dfd870715)

The mods I have to test this (Fabric 0.16.7 on MC 1.20.1)
![image](https://github.com/user-attachments/assets/8921b2a5-c6d4-4694-ab33-18e0b524eca7)

And finally, here's the log for when I click for the recipe of an item with this error:

``` [12:07:40] [Render thread/INFO]: [STDERR]: java.lang.NullPointerException: Cannot invoke "mezz.jei.api.gui.drawable.IDrawable.draw(net.minecraft.class_332)" because the return value of "mezz.jei.api.recipe.category.IRecipeCategory.getBackground()" is null
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//dev.emi.emi.jemi.JemiRecipe$JemiWidget.method_25394(JemiRecipe.java:167)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//dev.emi.emi.screen.RecipeScreen.method_25394(RecipeScreen.java:226)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//net.minecraft.class_437.method_47413(class_437.java:110)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//net.minecraft.class_757.method_3192(class_757.java:945)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//net.minecraft.class_310.method_1523(class_310.java:1219)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//net.minecraft.class_310.method_1514(class_310.java:802)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at knot//net.minecraft.client.main.Main.main(Main.java:250)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
[12:07:40] [Render thread/INFO]: [STDERR]: 	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

**Comment by Abbie5** [comment]
I can reproduce this error, here is a full log

[latest.log](https://github.com/user-attachments/files/17530367/latest.log)

**Comment by juh9870** [comment]
Seems to be caused by the same issue as #765

**Comment by emilyploszaj** [comment]
Fixed in 1.1.17
